sys/dev/intpm/intpm.c

```diff
@@ -879,7 +879,7 @@ intsmb_bread(device_t dev, u_char slave, char cmd, u_char *count, char *buf)
 		if (nread != 0 && nread <= SMBBLOCKTRANS_MAX) {
 			*count = nread;
```

Comment:
Do we need to do any validation that `nread <= *count` from entry to make sure we aren't overrunning `buf`?  (I'm unfamiliar with the smbus_bread API.)

Reply:
I think that `count` is an output parameter.
Callers are expected to handle up to `SMBBLOCKTRANS_MAX`  bytes of data.